src/platform_impl/linux/mod.rs

```diff
@@ -633,7 +634,7 @@ unsafe extern "C" fn x_error_callback(
 
 pub enum EventLoop<T: 'static> {
     #[cfg(feature = "wayland")]
-    Wayland(wayland::EventLoop<T>),
+    Wayland(Box<wayland::EventLoop<T>>),
```

Comment:
What is this box about?

Reply:
It complains that the size of the variant is too large.